Gemfile

```diff
@@ -38,12 +38,16 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'devise'
 gem 'will_paginate', '~> 3.1.0'
 gem 'rails-html-sanitizer'
-gem 'letsencrypt_plugin'
+#gem 'letsencrypt_plugin'
 
 gem 'whenever', :require => false
 
 gem 'coveralls', require: false
 
+# Gems for the project Mentor 
+
+gem 'stripe'
```

Comment:
@ArtOfCode- the idea would be building a community like StackOverflow, but where user can pay a monthly subscription and get mentor help to find the solution to their problems.. 

So stripes would be for paying a monthly subscription